src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -43,8 +52,37 @@ public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final En
         return false;
     }
 
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final List<ActionArgument> actionArgumentList,
+            @NotNull final Map<String, Object> argumentValueByActionArgumentNameMap) throws CommandException
+    {
+        boolean confirmDelete = false;
+        final Object confirmDeleteObject = argumentValueByActionArgumentNameMap.get("confirmDelete");
+        if (null != confirmDeleteObject)
+        {
+            Boolean confirmDeleteBoolean = (Boolean) confirmDeleteObject;
+            confirmDelete = confirmDeleteBoolean.booleanValue();
+        }
+        executeShared(server, sender, confirmDelete);
+    }
+
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        boolean confirmDelete = false;
+        if (args.length == 2)
+        {
+            if ("confirmDelete:".equalsIgnoreCase(args[0]))
+            {
+                if ("true".equalsIgnoreCase(args[1]))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)